use-cases/demos/object_detection/pytorch-yolo/entrypoint.sh

```diff
@@ -9,7 +9,28 @@
 cid=$(date +%Y%m%d%H%M%S%N)
 echo "cid: $cid"
 
-PROFILE_NAME="demo_pytorch_object_detection"
+#USE_ULTRALYTICS for using ultralytics implementation of YOLO object detection
+#Default set to 0, use torch hub implementation of YOLO object detection
+echo "USE_ULTRALYTICS: $USE_ULTRALYTICS"
 
-python3 object_detection/pytorch-yolo/yolov5_object_detection.py \
--m "$MQTT_HOSTNAME" -p "$MQTT_PORT" -t "$MQTT_TOPIC" -i "$INPUT_SRC" | tee >/tmp/results/r"$cid"_"$PROFILE_NAME".jsonl >(stdbuf -oL sed -n -e 's/^.*FPS: //p' | stdbuf -oL cut -d , -f 1 > /tmp/results/pipeline"$cid"_"$PROFILE_NAME".log)
+#INTEL_OPTIMIZED for using intel optimized packages of ultralytics implementation of YOLO object detection
+#Default set to 0, torch hub implementation doesn't support intel optimized packages
+echo "INTEL_OPTIMIZED: $INTEL_OPTIMIZED"
+
```

Comment:
based on docs, this INTEL_OPTIOMIZED is only meaningful when $USE_ULTRALYTICS = 1, so I would suggest moving this print out inside the if [ $USE_ULTRALYTICS -eq 1 ];  as out of the scope of =1, it doesn't mean anything

Reply:
done